lib/Folder/FolderMapping.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\GroupFolders\Folder;
+
+class FolderMapping {
```

Comment:
These classes should actually be entities with mappers, no?

Reply:
I guess so. I only made this change from array to class last minute, so I didn't give it much thought.

Reply:
Moved to https://github.com/nextcloud/groupfolders/issues/3389, I will revert these changes in this PR